registrations/tests/test_signup_group_get.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+from rest_framework import status
+
+from events.tests.utils import versioned_reverse as reverse
+from registrations.tests.factories import SignUpFactory, SignUpGroupFactory
+
+# === util methods ===
+
+
+def get_detail(api_client, signup_pk, query=None):
+    detail_url = reverse(
+        "signupgroup-detail",
+        kwargs={"pk": signup_pk},
+    )
+
+    if query:
+        detail_url = "%s?%s" % (detail_url, query)
+
+    return api_client.get(detail_url)
+
+
+def assert_get_detail(api_client, signup_pk, query=None):
+    response = get_detail(api_client, signup_pk, query)
+    assert response.status_code == status.HTTP_200_OK
+    return response
+
+
+# === tests ===
+
+
+@pytest.mark.django_db
+def test_admin_user_can_get_signup_group_without_signups(user_api_client, organization):
+    signup_group = SignUpGroupFactory(registration__event__publisher=organization)
+
+    response = assert_get_detail(user_api_client, signup_group.id)
+    assert len(response.json()["signups"]) == 0
+
+
+@pytest.mark.django_db
+def test_admin_user_can_get_signup_group_with_signups(user_api_client, organization):
+    signup_group = SignUpGroupFactory(registration__event__publisher=organization)
+    SignUpFactory(registration=signup_group.registration, signup_group=signup_group)
+    SignUpFactory(registration=signup_group.registration, signup_group=signup_group)
+
+    response = assert_get_detail(user_api_client, signup_group.id)
+    assert len(response.json()["signups"]) == 2
+
+
+@pytest.mark.django_db
+def test_regular_non_created_user_cannot_get_signup_group(
+    user_api_client, user, organization
+):
+    signup_group = SignUpGroupFactory(registration__event__publisher=organization)
+
+    organization.regular_users.add(user)
+    organization.admin_users.remove(user)
+
+    response = get_detail(user_api_client, signup_group.id)
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+def test_regular_created_user_can_get_signup_group(user_api_client, user, organization):
+    signup_group = SignUpGroupFactory(
+        registration__event__publisher=organization, created_by=user
+    )
+
+    organization.regular_users.add(user)
+    organization.admin_users.remove(user)
+
+    response = get_detail(user_api_client, signup_group.id)
+    assert response.status_code == status.HTTP_200_OK
+
+
+@pytest.mark.django_db
+def test__user_from_other_organization_cannot_get_signup_group(
```

Comment:
Trivial: not a fan of the double underscore and mixing both styles (`test_x_y_z` vs `test__x_y_z` inside a single file), but on the other hand it seems to be consistent with the codebase..